src/router.js

```diff
@@ -102,7 +102,7 @@ const routes = [
 ];
 
 const router = createRouter({
-  history: createWebHashHistory(),
+  history: createWebHistory(),
```

Comment:
This is the major change, the route will be inferred from the url now, and not the hash suffix.